sprs/Cargo.toml

```
@@ -21,7 +21,7 @@ multi_thread = ["dep:rayon", "dep:num_cpus"]
 
 [dependencies]
 num-traits = "0.2.0"
-ndarray = ">=0.15, <=0.16"
+ndarray = ">=0.15, <=0.16.1"
```

Same suggestions as in comment above:

```suggestion
ndarray = ">=0.15.0, <0.17"
```

```suggestion
ndarray = "0.16"
```